python/paddle/fluid/data_feeder.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, place, lod_level, shape, dtype):
         self.lod = []
 
         for i in six.range(lod_level):
```

Comment:
`lod_level` describes the depths of the lod info.
so it may be renames to something like `sequence_lengths_depth`
Again, this can be done in the next PR.